### DISCUSSION TITLE
Is there a way to delete opened discussion in community?

### DISCUSSION THREAD

**Discussion by JohnDev297** [community/community#47188] [Other Feature Feedback, Questions, & Ideas] [answered]
I'm trying to delete a discussion but can't find any delete discussion button?

**Comment by jge162** [comment, marked as answer]
It has to be your repo/you are the owner to delete a discussion.

**Reply by JohnDev297** [reply]
Alright thanks.

**Reply by 3ggr2025** [reply]
https://github.com/orgs/community/discussions/47188#discussioncomment-4952956
